public/js/view.js

```diff
@@ -94,78 +94,122 @@ function makeSVG(w, h) {
 
 
 
+/********************************** Drawing ***********************************/
+// HTML Div
+var canvas = $("#canvas");
+var W = 600, H = 600;
+canvas.css("height", H);
+canvas.css("width", W);
+
+// HTML SVG object
+var svg = $(makeSVG(W, H));
+
+// Measurements for drawing purposes
+var scale, radius, offset;
+
+
+
 /**
- * This function takes a GameSpace object and renders the board
- * on the HTML page.
+ * This function initializes all that are needed for drawing.
+ * 1. Calculates necessary variables from the size of the board.
+ * 2. Puts the svg object inside the canvas div
  * 
- * Precondition: The HTML page must include a div element with "canvas" as its id.
+ * @param {type} board - a GameSpace instance
+ */
+function init(board) {
+    // 1. Variables
+    scale = W / board.size;
+    radius = (scale / 2) - 1;
+    offset = scale / 2;
+
+    // 2. append the svg object to the canvas div.
+    canvas.append(svg);
+}
+
+/**
+ * This function draws any given board state i.e. a GameSpace object.
+ * This function:
+ * 1. Extracts the board state from the GameSpace object.
+ * 2. Clears the svg object. This is done to prevent duplicate sub-svg objects
+ *      (i.e. lines and previously placed stones)
+ * 3. Draw lines
+ * 4. Draw stones
  * 
  * @param {object} board - A GameSpace object
  */
 function draw(board) {
 
-    // Extracting the array representation of the board
+    // 1. Extracting the board representation
     board = board.board.board;
```

Comment:
Also, consider changing the variable name to indicate that it is a private variable. So in the future, I will know which variables are to be used with getters and setters.